src/platform/core/common/styles/_basicElements.scss

```diff
@@ -19,56 +19,56 @@
   font-family: 'Roboto';
   font-style: normal;
   font-weight: 300;
-  src: local("Roboto Light"), local("Roboto-Light"), url("../../../../../roboto-fontface/fonts/roboto/Roboto-Light.ttf") format("truetype");
+  src: local("Roboto Light"), local("Roboto-Light"), url("../../../../../roboto-fontface/fonts/roboto/Roboto-Light.woff2") format("woff2");
```

Comment:
I think it would make sense to expose these URL paths as a variable with a `!default` value. That would give anyone the opportunity to override these paths from outside.

Reply:
I created https://issues.apache.org/jira/browse/NIFI-6291 to address this issue